### DISCUSSION TITLE
Should We Shift Our Focus to P2P Economies?

### DISCUSSION THREAD

**Discussion by ted-livingston** [kinfoundation/kin-rfcs#153] [Spend Based KRE]
One thing I’ve been thinking about is if “traditional spends” are the right pursuit for Kin, or if there might be a better place to focus our energy.

The problem I see with focusing on “traditional spends” is that existing payment methods already work well. If you’re Clash of Clans or Roblox or Amazon you already have a great payment experience and you’re already making a lot of money. Why would you switch your payments to a cryptocurrency? It would be more friction for your users, more risk with Apple/Google, and less revenue for you. Yes you could offer it as “just another payment option”, but that would increase user friction which would likely reduce your overall revenue.

Instead I wonder if we should be focusing on experiences where existing payment mechanisms don’t work. What are those experiences? To me it is one where users both give *and* receive payments. Existing payments mechanisms are great for taking money from users, but almost none of them allow you to pay money back to users.

I wonder if this is where we should focus our energy as a community: payment experiences where users transact with each other. I’m tentatively calling these “P2P economies”. So for example, in P365 we don't ask users to buy beauty products with Kin, as existing payment mechanisms work just fine for that. Instead we ask one user to pay Kin to another user to get their beauty advice. Or in Madlipz a user doesn't pay to get a feature in the app, but perhaps one user pays another user to join their exclusive jokes community inside the app, where they can get exclusive content and connect with other users. Tapatalk could do something similar. And in many ways PeerBet already has.

This would create demand because a subset of power users would be charging in Kin (no other payment mechanism allows them to receive payments from a global audience) which would require the rest of the users to buy Kin in order to be able to enjoy those experiences. Yes most of this purchased Kin would flow to the “creators”, much of which they might sell. But a lot of bought Kin would be left over and accumulate in the wallets of all the other users, and creators would likely keep some as well.

In this way perhaps Kin could go after experiences that are hard to provide with existing payment mechanisms, rather than trying to compete head on for experiences where existing payment mechanisms already work well.

What do you all think?

**Comment by Freedom-kin** [comment]
Hi Ted, I am somewhat surprised to read this.   
It seems you give all kinds of reasons why NOT to implement Kin.

The first questions that come to mind are:   
- how does the KRE fit into P2P payments?   
- Is everyone eligible for KRE (not just developers)?   
- Should we then reward the balances of users, like now the AUB?   
- And since all this does not really lead to (traditional) economic growth,   
  **who will pay for the KRE**, since free money does not exist.

But to return to your original question about why a company would want to integrate Kin as (part) of a payment system.

The proposal to reward SPENDS with **a 30% reward** seems to me to be an excellent incentive. Where do you get a 30% bonus on top of your net turnover?

I don't expect a company to fully switch to Kin for all payments, but if it has a good revenue model, it can expand its model by doing (part of) the same in the Kin Economy.

It seems necessary to me that we design a system that is also sustainable in the long term. This means that the system should not be completely dependent on KRE.

KRE will decrease every year (according to your model) but more and more apps are added. Even now, only a few developers can make a living from it. What if there are 50 or more? So there MUST be a revenue model next to the KRE.

If there are ways to game a KRE based on SPENDS, let's look for ways to prevent it.

And quite frankly...we now also have a system that is easy to game, and yet we accept that....a system that does virtually nothing for the growth of our economy and only causes inflation.

So even if there's a way to game a KRE based on SPENDS, and we can't prevent it 100%, that's still better because at least it stimulates growth AND it is sustainable.
.

**Comment by amiraliaaa** [comment]
I’m glad you brought this up @ted-livingston . I really like the idea of focusing on P2P transactions as the main parameter in the KRE formula.  

One of the main issues with the current KRE and the spend-based one is that developers need to fund the users but they need to make sure the balance of earns to spends makes sense otherwise they wont make any profits (i.e. they give away X amount of Kin in earns to users but receive Y Kin which could be a lot less than X and its hard to calculate it). Focusing on P2P transactions solves this problem and developers know the Kin that users are spending gets transferred to other users and therefore they can have a better and more clear plan for designing the Kin economy in their apps. 

Focusing on P2P transactions creates better user experience and encourages user to user interactions which helps to build the economy.  

Finally it's harder to game this model as the two sides of the transaction in P2P transactions are not the developer unlike spend and earn transactions. 

  I like the idea of focusing on P2P transactions as the main parameter. We just need to figure out the details. Some questions that we need to think about:
  - Are we focusing on number of P2P transactions OR the total amount OR a combination of these two? 
  - I still think we need to somehow incorporate Active Users in the formula maybe as a cap as we have right now. The Active Users definition changes to users who have done a P2P transaction in the past month? Or 3 in the past month?
  - Do we want to keep AUB as one of the parameters?

**Reply by Freedom-kin** [reply]
@amiraliaaa

You write

> One of the main issues with the current KRE **and the spend-based one** is that developers need to fund the users but they need to make sure the balance of earns to spends makes sense otherwise they wont make any profits (i.e. they give away X amount of Kin in earns to users but receive Y Kin which could be a lot less than X and its hard to calculate it). 

In the spend-based KRE developers do not need to fund the users.
We have to go to a point that users buy Kin themselves.
Funding users does not grow the value of the Kin Economy.
Just giving it away brings more inflation.

Of course you can give away some Kin, as a promotion, but as you said, spends need to be more than earns. In fact, that is what all businesses do, having more sales then costs.  It's not hard to calculate.

P2P transactions is what previous iterations of KRE got in trouble
On the blockchain it's no different than a withdrawal/deposit.
Meaning that if you reward P2P you also reward withdrawals.
We have seen how that was gamed.
So, from a technical standpoint P2P seems out of the question.

Spends and earns however can be clearly distinguished from the rest.

**Reply by amiraliaaa** [reply]
@Freedom-kin 

We definitely need to move to a point where users buy Kin themselves but we are not at that point right now and for the next iteration of the KRE the developers still need to fund users. Buying kin by users also doesnt go against a P2P based KRE. It actually helps the economy as more users have Kin to send to other users.

I am one of the developers and I can tell you estimating the balance between earns and spends has been super challenging specially with the delay in the reporting and other issues. Relying on P2P solves this problem once and for all as developers create experiences for users to interact with each other without worrying about the spends/earns.

I disagree with you on P2P transactions creating troubles for the KRE. Its actually the opposite. Spends are creating problems and they are SUPER easy to be gamed. Cause for spend you just need to create random wallets and transfer the amounts from those wallets into your own wallet. However when dealing with P2P transactions in order to game the system you need to create wallets for both the senders and the recipients and then somehow transfer everything into one wallet which makes the process a lot harder than the spend based KRE.

If your concern is deposit/withdrawals then maybe we can add a constraint that both sender and the recipient's wallets need to be created on the same app or one of the ecosystem apps if we wanna allow P2P transfers between apps. 

I really like the idea of having a P2P-based KRE as it really helps the Kin economy to grow.

**Reply by Polype01** [reply]
Hi Amir 

Focusing on P2P transactions solves this problem and developers know the Kin that users are spending gets transferred to other users and therefore they can have a better and more clear plan for designing the Kin economy in their apps.

Regarding this, it is ok too consider more P2P transactions but it would be also easy to game. How about the following pattern fake user account A transferring 2500 KIN to user B, transferring 2500 KIN to user C…
There could be a more elaborated pattern, more difficult to identify. That’s why I think considering P2P transactions is good but P2App transactions need still to be considered

**Comment by KinLover2** [comment]
I think moving towards a more p2p heavy model has many benefits. I don't claim to know how it can be integrated into the calculations for KRE, but p2p allows developers to create amazing experiences that's unique to Kin Ecosystem. Many "activities" on various platforms and apps are just "feel good" rewards, liking, commenting, sharing, etc. Kin allows these behaviours to be part of a win-win-win scenario where both parties benefit on the same side of the developers. I believe at some point p2p transactions were the main category for calculating KRE and moving towards that would be beneficial as it will also provide an easier way to determine how much a Kin experience is worth. A "like" on any platform is solely dependant on the size of the audience and value of the network as opposed to arbitrary values given by the developers who'd need to adjust it as KRE formula develops. Promoting real interactions and transactions would ultimately be the only way Kin can be successful, we need to think about the "why" of needing Kin as opposed creating "buy pressure" they might and are hand in hand, but looking at utility first, making it inevitable part of daily digital life of people is how we can guarantee long term success of the project.

**Comment by skyf0xx** [comment]
I would say the current incentive (AUB) or any of its derivatives focuses on p2p. The best way developers can earn is by keeping the trading between users and not user to app.

I would say the AUB metric is the best one we've had so far.

**Reply by Freedom-kin** [reply]
Hi Wil, I think if the sole purpose is to split a bag of money between developers you are absolutely right. However, if the goal is to build an economy, you can't get a worse metric than the AUB. Try answering the following questions: **Who pays for the KRE payouts?...and what do those who pay for them get in return?**

**Reply by skyf0xx** [reply]
If the KRE was only inflating 1,000 Kin/ day - developers/ users would be paying for any extra Kin going into the economy. Developers would naturally offset this payment to users, though buy modules and keep the 1,000 Kin for themselves as a reward.

I totally agree that at this point in time, Kin is just flowing freely into the economy.
